.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
@Zizzamia @codynova which version of yarn you're using in this repo? I'm getting v1 migrations, it's causing some local git changes after runnning `yarn install` (check last commit 794b6c0)

Reply:
My local dev setup:

```
$ yarn -v
3.4.1
```

Reply:
Nevermind, I pushed some changes in ca29ffb to explicitly use yarn v1 when working in this repo...

```
$ yarn -v
1.22.19
```